package.json

```
@@ -101,10 +102,10 @@
     "webpack": "^5.52.1"
   },
   "peerDependencies": {
-    "ember-source": "*"
+    "ember-source": ">= 4"
```

should we make this
```suggestion
    "ember-source": ">= 4.8"
```
as we say 4.8+ in readme and only test those versions?

---

applied